addon-test-support/teardown-rendering-context.js

```diff
@@ -0,0 +1 @@
+export default function() {}
```

Comment:
guess this is for future extensibility? should this call `ctx.clearRender()`?

Reply:
ya, it got content in a later commit